huaweicloud/utils/type_convert.go

```diff
@@ -105,7 +105,7 @@ func ValueIngoreEmpty(v interface{}) interface{} {
 		return nil
 	}
 
-	if (vl.Kind() == reflect.Array || vl.Kind() == reflect.Slice) && vl.Len() == 0 {
+	if (vl.Kind() == reflect.Array || vl.Kind() == reflect.Slice || vl.Kind() == reflect.Map) && vl.Len() == 0 {
```

Comment:
looks empty map will be ignored in utils.RemoveNil,
so this change is not necessary

Reply:
sure, and this commit has been removed.